docs/installation.md

```diff
@@ -19,7 +19,7 @@ The following instructions are for setting up a version of Amundsen using Docker
     If it's your first time, you may want to proactively go through [troubleshooting](troubleshooting) steps, especially the first one related to heap memory for ElasticSearch and Docker engine memory allocation (leading to Docker error 137).
 4. Ingest provided sample data into Neo4j by doing the following: _(Please skip if you are using Atlas backend)_
 
-   * In a separate terminal window, change directory to the [amundsendatabuilder](https://github.com/amundsen-io/amundsendatabuilder) submodule.
+   * In a separate terminal window, change directory to [databuilder](https://github.com/amundsen-io/amundsen/tree/main/databuilder).
```

Comment:
does it cover all obsolete entries for databuilder repo?

Reply:
Looks like there are many more:
```
fokkodriesprong@MacBook-Pro-van-Fokko amundsen % grep -R "amundsendatabuilder" . | wc -l
     337
```

Reply:
Looks like about 1/2 of those are from CHANGELOGs and are correct, but the rest do need to be fixed